lib/modules/manager/mint/readme.md

```diff
@@ -0,0 +1,14 @@
+Renovate supports updating Mintfiles.
+
+[`yonaskolb/Mint` repository on GitHub](https://github.com/yonaskolb/Mint)
+
+You must put the library version in the Mintfile:
+
+```
+// Good:
+SwiftGen/SwiftGen@6.6.1
+realm/SwiftLint @ 0.48.0
```

Comment:
> realm/SwiftLint @ 0.48.0

Are you sure that you can put spaces between the package name and the version number? It looks wrong to me, but maybe this is allowed syntax in a Mintfile. 😄

Reply:
Hi! Sorry for the late reply 🙇‍♂️
As you say, Mintfile allows its syntax.
Here is the test case. (https://github.com/yonaskolb/Mint/blob/master/Tests/Fixtures/Mintfile#L4)